newrelic.yml

```diff
@@ -169,6 +169,9 @@ common: &default_settings
   # If true, disables Action Cable instrumentation.
   # disable_action_cable_instrumentation: false
 
+  # If true, disables ActionDispatch instrumentation
```

Comment:
Please add a space between "Action" and "Dispatch"

Reply:
I was missing the ending period (.) as well. 😞 Addressed with [e4f9e0d](https://github.com/newrelic/newrelic-ruby-agent/pull/1745/commits/e4f9e0d6dcd952490c9345daa8471c4b55ae637e)